common/libs/exception_thread/ex_thread.py

```diff
@@ -2,10 +2,10 @@
 import traceback
 import queue
 from multiprocessing import TimeoutError
-import logging
```

Comment:
As I told. I don't like this solution. wat about replacing for lobs  -logger = logging.getLogger("thread") to common.scaf.get_logger(__name__) under the hood?

Reply:
will be fixed in next PR